File: DKImagePickerControllerDemo_OC/DKImagePickerControllerDemo_OC/CustomCameraUIDelegate/CustomCameraUIDelegate.m
//
//  CustomCameraUIDelegate.m
//  DKImagePickerControllerDemo_OC
//
//  Created by zm on 2017/7/7.
//  Copyright © 2017年 zm. All rights reserved.
//

#import "CustomCameraUIDelegate.h"
#import "CustomCamera.h"
@implementation CustomCameraUIDelegate
- (UIViewController *)imagePickerControllerCreateCamera:(DKImagePickerController *)imagePickerController  {
    
    UIViewController * picker = [CustomCamera new];
    return picker;
}
@end

File: DKImagePickerControllerDemo/CustomCameraUIDelegate/CustomCameraUIDelegate.swift
//
//  CustomCamera.swift
//  DKImagePickerControllerDemo
//
//  Created by ZhangAo on 16/3/17.
//  Copyright © 2016年 ZhangAo. All rights reserved.
//

import UIKit

open class CustomCameraUIDelegate: DKImagePickerControllerDefaultUIDelegate {
	
    open override func imagePickerControllerCreateCamera(_ imagePickerController: DKImagePickerController) -> UIViewController {
        let picker = CustomCamera()
        picker.sourceType = .camera
        
        return picker
    }
    
}
